drivers/serial/uart_ngultra.h

```diff
@@ -0,0 +1,23 @@
+#ifndef __UART_NX_NGULTRA_H__
+#define __UART_NX_NGULTRA_H__
+
+#define UART_DATA          0x0
```

Comment:
I would just put these #defines in the `.c` file.

Reply:
Done